docs/trino-connector/development.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Gravitino connector development"
+slug: /trino-connector/development
+keyword: gravitino connector development 
+license: "Copyright 2024 Datastrato Pvt Ltd.
+This software is licensed under the Apache License version 2."
+---
+
+This document is to guide users through the development of the Gravitino connector for Trino locally.
+
+## Prerequisites
+
+Before you start developing the Gravitino trino connector, you need to have the following prerequisites:
+
+1. You need to start the Gravitino server locally, for more information, please refer to the [start Gravitino server](../how-to-install.md)
+2. Create a catalog in the Gravitino server, for more information, please refer to the [Gravitino metadata management](../manage-metadata-using-gravitino.md). Assuming we have just created a MySQL catalog using the following command:
+
+```curl
+curl -X POST -H "Content-Type: application/json" -d '{"name":"test","comment":"comment","properties":{}}' http://localhost:8090/api/metalakes
+
+curl -X POST -H "Content-Type: application/json" -d '{"name":"mysql_catalog3","type":"RELATIONAL","comment":"comment","provider":"jdbc-mysql", "properties":{
+  "jdbc-url": "jdbc:mysql://127.0.0.1:3306?useSSL=false&allowPublicKeyRetrieval=true",
+  "jdbc-user": "root",
+  "jdbc-password": "123456",
+  "jdbc-driver": "com.mysql.cj.jdbc.Driver"
+}}' http://localhost:8090/api/metalakes/test/catalogs
+```
+
+:::note
+Please change the above `localhost`, `port` and the names of metalake and catalogs accordingly.
+:::
+
+
+## Development environment
+
+To develop the Gravitino connector locally, you need to do the following steps:
+
+### IDEA
+
+1. Clone the Trino repository from the [GitHub](https://github.com/trinodb/trino) repository. We advise you to use the release version 426 or 435. 
+2. Open the Trino project in your IDEA.
+3. Create a new module for the Gravitino connector in the Trino project as the following picture (we will use the name `trino-gravitino` as the module name in the following steps). ![trino-gravitino](../assets/trino/create-gravitino-connector.jpg)
```

Comment:
How to create a new module

Reply:
This should be tiny things compared to developing The Trino connector. If users lack this knowledge, I don't think they can develop the connector effectively.